### DISCUSSION TITLE
"Did you like this map?" prompt on completing beatmap

### DISCUSSION THREAD

**Discussion by ozornayashoujo** [ppy/osu#13015] [Ideas]
At first I thought about user rating being too wide (10-star system), but that problem had already been brought up. Then I thought about why people might not rate a map: quickly closing a map after playing, deleting maps, etc. But often people visit their own page to see their progress.

What about a "Did you like this map?" question near every top play, and every recent play? That seemed the most logical to me. You could also suggest to rate random recent plays as pop-ups of some sort. I don't care whether the system will be 3-rating (like/neutral/dislike) or 2-rating, but the idea to _casually_ remind people to rate maps is very attractive to me.

The result should be more accurate, more player-oriented user rating, which is definetly better than the current system

**Comment by abraker95** [comment]
I am against this being a popup as it may cause annoyance. I suggest placing it close to the back button in the results screen where it would be hard to not ignore. Currently you have to scroll down all the way down to see it, so only a person who goes out of their way to decide to rate a map is going to go to that relatively remote area to do so. Putting it closest to the elements the user interacts with lowers the effort required to navigate toward it. Additionally, I suggest designing it such it draws the user's attention but not so much that it is annoying.

I think a 2-rating (Like/Dislike) is sufficient enough. If the person is neutral then the person wouldn't rate it.

**Comment by HoutarouOreki** [comment]
I too think it should not be an intrusive reminder, don't we all hate those things?

**Comment by ozornayashoujo** [comment]
> I too think it should not be an intrusive reminder, don't we all hate those things?

I completely agree. The idea I wanted to convey is that there are many ways to go about raising feedback level and quality, maybe even try something completely new, if possible.
Thanks for the feedback
